test/header/cache-control.test.ts

```diff
@@ -29,6 +23,12 @@ describe('test Cache-Control header', () => {
 
     // 10 Seconds in milliseconds
     expect(result).toBe(10 * 1000);
+
+    const mustRevalidate = defaultHeaderInterpreter({
```

Comment:
this check doesn't make sense, `must-revalidate` just prevent the storage value become stale.